demos/logging-stack/logging_stack.h

```diff
@@ -36,15 +35,38 @@
 #include <stdint.h>
 #include <string.h>
 
+/* The macro definition for LIBRARY_LOG_NAME is for Doxygen
+ * documentation only. This macro is typically defined in only the
+ * <library>_config.h file or the demo_config.h file. */
+
+/**
+ * @brief The name of the library or demo to add as metadata in log messages
+ * from the library or demo.
+ *
+ * This metadata aids in identifying the module source of log messages.
+ * The metadata is logged in the format `[ <LIBRARY-NAME> ]` as a prefix to the
+ * log messages.
+ * Refer to #LOG_METADATA_FORMAT for the complete format of the metadata prefix in
+ * log messages.
+ */
+#ifdef DOXYGEN
+    #define LIBRARY_LOG_NAME    "<LIBRARY_NAME>"
```

Comment:
Have you generated the doxygen for this? I'm getting errors from this file:
```
logging_stack.h:92: warning: Member LogError(message) (macro definition) of file logging_stack.h is not documented.
logging_stack.h:93: warning: Member LogWarn(message) (macro definition) of file logging_stack.h is not documented.
logging_stack.h:94: warning: Member LogInfo(message) (macro definition) of file logging_stack.h is not documented.
logging_stack.h:95: warning: Member LogDebug(message) (macro definition) of file logging_stack.h is not documented.
```

Reply:
Added documentation for the macros